app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,20 @@
+class LikesController < ApplicationController
+  def destroy
+    likes = current_user.likes.where(post_id: like_params[:post_id])
+    likes.each{|l| l.destroy}
+    redirect_to blog_path(like_params[:blog_id])
+  end
+
+  def new
+    Like.new(
+      user: current_user,
+      post: Post.find(like_params[:post_id])
+    ).save
+    redirect_to blog_path(like_params[:blog_id])
+  end
```

Comment:
You wouldn't usually have a `new` action actually create an object. That's usually reserved for the `create` action. 

And if you're going to save an object right after initialising it without any kind of in between steps, you might as well use `Like.create( ... )` rather than `Like.new( ... ).save`!

Reply:
Whoops, thanks. I even had the keyword create in like_spec!